### DISCUSSION TITLE
Make new/update lists on homepage more dynamic

### DISCUSSION THREAD

**Discussion by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#1383] [Ideas]
It's always slightly bothered me that new packages and updates on the homepage drop off the list beyond our cut-off without any means to page through older results. This can happen fast if a whole batch of packages are added.

We could keep a record and add paging but there are reasons we've avoided keeping that sort of state in the database so far.

What if we leaned into a service that shows events in a timeline? We're already posting all our additions and updates to the `@SwiftPkgUpdates` feed on twitter.

We could
- add a [link to a search](https://twitter.com/search?q=from%3ASwiftPkgUpdates%20⬆%EF%B8%8F&src=typed_query&f=live) to the end of the two lists *)
- embed tweets from new/updates searches instead of the columns

One nice benefit from this would be making the update feed more discoverable.

*) there's a problem searching for the `📦` in our "new package" tweets. I can't find any tweets with that emoji in it and I suspect it's due to the emoji. We may need to replace it. I've added an issue tracking that: #1384

**Comment by daveverwer** [comment]
I was about to write that we have enough unique text in the tweet to do a search without the emoji as we have "just released" and "just added" as unique between the two types, but it's not that simple unfortunately.

Searching for all of these things returns no results:

* `from:SwiftPkgUpdates just added`
* `from:SwiftPkgUpdates "just added"`
* `from:SwiftPkgUpdates added`
* `from:SwiftPkgUpdates "added"`
* `from:SwiftPkgUpdates vcomponentkit` (our latest added package)

I can't really explain it, and it's not related to updated/added tweets either because `from:SwiftPkgUpdates Crossroad` (our latest updated package) also returns nothing.

I think if we want to have more packages on the front page (which I agree with), we should do it as part of the front page rather than linking out to Twitter.

As a slight aside, we definitely shouldn't embed tweets. The tweet embedding JavaScript you need from Twitter is their way of getting their analytics tracker into every site on the web and isn't something I want to support.
